format.md

```diff
@@ -192,6 +192,16 @@ however, implementations must be able to ingest arrays with other names when the
 interpretation is unambiguous (e.g., for xy and xyzm interleaved coordinate
 interpretations).
 
+### List types
+
+Arrow has multiple list types, including `List` (parameterized by int32 offsets), `LargeList` (parameterized by int64 offsets), and newer `ListView` and `LargeListView`.
+
+Implementations SHOULD accept `LargeList` int64 offset buffers but MAY produce only `List` int32 offset buffers.
+
+The `List` type will not overflow until there are `2^31 + 1` entries in the coordinates array. For two 8-byte floats, this would require 32GB of memory in a single coordinates array, and is thus unlikely to occur often in practice.
+
+`ListView` and `LargeListView` are not yet well supported throughout the ecosystem, but implementations SHOULD accept these types as well.
```

Comment:
For this part, do known implementations currently do that? 

I know geopandas for example does not do that, and I see we should actually add some validation (with a proper error message) for that case? And I am not sure any of the geoarrow-c / geoparrow-pyarrow packages support ingesting those?

Reply:
Yeah I didn't know what I should write about ListView/LargeListView.

I don't believe ListView/LargeListView are fully operational in arrow-rs, so geoarrow-rs doesn't support them as input.

Should we just take this sentence out?

Reply:
If we don't know of anyone potentially producing it or even supporting it when receiving data, I would personally just leave it out for now

Reply:
removed